core/templates/dev/head/components/summary_tile/ExplorationSummaryTileDirective.js

```diff
@@ -112,14 +112,21 @@ oppia.directive('explorationSummaryTile', [function() {
         };
 
         $scope.getLastUpdatedDatetime = function() {
+          if (!$scope.getLastUpdatedMsec()) {
+            return false;
```

Comment:
return null, not false. Null indicates absence of a value. False indicates that the return value is usually boolean.